### ISSUE TITLE
cc1352p-2 cannot be found - addon is not starting

### ISSUE THREAD

**Issue by FredD35** [danielwelch/hassio-zigbee2mqtt#492]
Hi all,

I’m running

•	HassioVM from here: https://www.home-assistant.io/hassio/installation/
•	Mosquitto broker 5.1
•	Zigbee2mqtt 1.15.0 (or the dev branch as well)

I had my system up and running fine, using a CC2531 stick. But I had to replace the stick and bought a cc1352p-2 coordinator. I was able to flash the current firmware from here: https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin

After starting the zigbee2mqtt addon I get the following error message:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
./run.sh: line 17: [Info] Configuration backup found in /share/zigbee2mqtt/.configuration.yaml.bk. Skipping config backup.: No such file or directory
[Info] Socat is DISABLED and not started
2020-12-08T17:27:57: PM2 log: Launching in no daemon mode
2020-12-08T17:27:57: PM2 log: App [npm:0] starting in -fork mode-
2020-12-08T17:27:57: PM2 log: App [npm:0] online
> zigbee2mqtt@1.15.0 start /zigbee2mqtt-1.15.0
> node index.js
Zigbee2MQTT:info  2020-12-08 17:27:59: Logging to console and directory: '/share/zigbee2mqtt/log/2020-12-08.17-27-59' filename: log.txt
Zigbee2MQTT:debug 2020-12-08 17:27:59: Removing old log directory '/share/zigbee2mqtt/log/2020-12-08.17-27-28'
Zigbee2MQTT:debug 2020-12-08 17:27:59: Loaded state from file /share/zigbee2mqtt/state.json
Zigbee2MQTT:info  2020-12-08 17:27:59: Starting Zigbee2MQTT version 1.15.0 (commit #1.15.0)
Zigbee2MQTT:info  2020-12-08 17:27:59: Starting zigbee-herdsman...
Zigbee2MQTT:debug 2020-12-08 17:27:59: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null},"backupPath":"/share/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/share/zigbee2mqtt/database.db.backup","databasePath":"/share/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6758},"serialPort":{"path":"/dev/ttyACM0"}}'
Zigbee2MQTT:error 2020-12-08 17:27:59: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2020-12-08 17:27:59: Failed to start zigbee
Zigbee2MQTT:error 2020-12-08 17:27:59: Exiting...
Zigbee2MQTT:error 2020-12-08 17:27:59: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyACM0'
    at Znp.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:136:32)
    at Generator.next (<anonymous>)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:23:12)
    at SerialPort.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:134:49)
    at SerialPort._error (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:198:14)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:242:12
npm 
ERR!
 code ELIFECYCLE
npm ERR! errno 1
npm
 ERR! zigbee2mqtt@1.15.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.15.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-12-08T16_27_59_900Z-debug.log
2020-12-08T17:28:01: PM2 log: App [npm:0] exited with code [1] via signal [SIGINT]
2020-12-08T17:28:01: PM2 log: App [npm:0] starting in -fork mode-
2020-12-08T17:28:01: PM2 log: App [npm:0] online
> zigbee2mqtt@1.15.0 start /zigbee2mqtt-1.15.0
> node index.js
Zigbee2MQTT:info  2020-12-08 17:28:02: Logging to console and directory: '/share/zigbee2mqtt/log/2020-12-08.17-28-02' filename: log.txt
Zigbee2MQTT:debug 2020-12-08 17:28:02: Removing old log directory '/share/zigbee2mqtt/log/2020-12-08.17-27-31'
Zigbee2MQTT:debug 2020-12-08 17:28:02: Loaded state from file /share/zigbee2mqtt/state.json
Zigbee2MQTT:info  2020-12-08 17:28:02: Starting Zigbee2MQTT version 1.15.0 (commit #1.15.0)
Zigbee2MQTT:info  2020-12-08 17:28:02: Starting zigbee-herdsman...
Zigbee2MQTT:debug 2020-12-08 17:28:02: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null},"backupPath":"/share/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/share/zigbee2mqtt/database.db.backup","databasePath":"/share/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6758},"serialPort":{"path":"/dev/ttyACM0"}}'
Zigbee2MQTT:error 2020-12-08 17:28:02: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2020-12-08 17:28:02: Failed to start zigbee
Zigbee2MQTT:error 2020-12-08 17:28:02: Exiting...
Zigbee2MQTT:error 2020-12-08 17:28:02: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyACM0'
    at Znp.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:136:32)
    at Generator.next (<anonymous>)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:23:12)
    at SerialPort.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:134:49)
    at SerialPort._error (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:198:14)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:242:12
npm
 ERR!
 code ELIFECYCLE
npm ERR! errno 1
npm 
ERR! zigbee2mqtt@1.15.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.15.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

I’ve read that I must change the pan_id, so I increased this number by 1 – actually not sure what I was doing, and it didn’t change anything.
The error message pointed me to the USB port and I tried to figure out, whether I have to change this value, but now I’m lost. Entering the value ‘null’ to let the system automatically detect the right port does not work, because zigbee2mqtt blocks me from saving the config file. 

![Screen](https://user-images.githubusercontent.com/75689929/101516355-aeba3200-397f-11eb-984d-1605635bce83.PNG)
Actually, when I use the console to find the USB devices, it does not even list a ttyACM0. Could somebody give me a hint on how to proceed?

Thanks in advance,
Fred

**Comment by ciotlosm** [comment]
You error is `cannot open /dev/ttyACM0`. You can use HassOS > Supervisor > System > Host System > ┇> Hardware and see what you have there under `serial`.

**Comment by FredD35** [comment]
Thanks for answering.

I followed your steps and got this:

![Screenshot 2020-12-09 093103](https://user-images.githubusercontent.com/75689929/101605641-ba9b0800-3a02-11eb-8629-bc31f35f7d14.png)

Afterwards I tried all 4 options and got the same result:

`
Zigbee2MQTT:error 2020-12-09 09:37:04: Error: Error while opening serialport 'Error: Error: I/O error setting custom baud rate of 115200'
    at Znp.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:136:32)
    at Generator.next (<anonymous>)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:23:12)
    at SerialPort.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:134:49)
    at SerialPort._error (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:198:14)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:242:12
npm 
ERR!
 code ELIFECYCLE
npm ERR! errno 1
npm 
ERR!
 zigbee2mqtt@1.15.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.15.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm 
ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-12-09T08_37_04_520Z-debug.log
2020-12-09T09:37:06: PM2 log: App [npm:0] exited with code [1] via signal [SIGINT]
2020-12-09T09:37:06: PM2 log: App [npm:0] starting in -fork mode-
2020-12-09T09:37:06: PM2 log: App [npm:0] online
> zigbee2mqtt@1.15.0 start /zigbee2mqtt-1.15.0
> node index.js
Zigbee2MQTT:info  2020-12-09 09:37:07: Logging to console and directory: '/share/zigbee2mqtt/log/2020-12-09.09-37-07' filename: log.txt
Zigbee2MQTT:debug 2020-12-09 09:37:07: Removing old log directory '/share/zigbee2mqtt/log/2020-12-09.09-36-35'
Zigbee2MQTT:debug 2020-12-09 09:37:07: Loaded state from file /share/zigbee2mqtt/state.json
Zigbee2MQTT:info  2020-12-09 09:37:07: Starting Zigbee2MQTT version 1.15.0 (commit #1.15.0)
Zigbee2MQTT:info  2020-12-09 09:37:07: Starting zigbee-herdsman...
Zigbee2MQTT:debug 2020-12-09 09:37:07: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null},"backupPath":"/share/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/share/zigbee2mqtt/database.db.backup","databasePath":"/share/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6758},"serialPort":{"path":"/dev/ttyS3"}}'
Zigbee2MQTT:error 2020-12-09 09:37:07: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2020-12-09 09:37:07: Failed to start zigbee
Zigbee2MQTT:error 2020-12-09 09:37:07: Exiting...
Zigbee2MQTT:error 2020-12-09 09:37:07: Error: Error while opening serialport 'Error: Error: I/O error setting custom baud rate of 115200'
    at Znp.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:136:32)
    at Generator.next (<anonymous>)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:23:12)
    at SerialPort.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:134:49)
    at SerialPort._error (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:198:14)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:242:12
npm 
ERR!
 code ELIFECYCLE
npm ERR! errno
 1
npm 
ERR! zigbee2mqtt@1.15.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.15.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-12-09T08_37_07_774Z-debug.log
2020-12-09T09:37:09: PM2 log: App [npm:0] exited with code [1] via signal [SIGINT]
2020-12-09T09:37:09: PM2 log: App [npm:0] starting in -fork mode-
2020-12-09T09:37:09: PM2 log: App [npm:0] online
> zigbee2mqtt@1.15.0 start /zigbee2mqtt-1.15.0
> node index.js
Zigbee2MQTT:info  2020-12-09 09:37:10: Logging to console and directory: '/share/zigbee2mqtt/log/2020-12-09.09-37-10' filename: log.txt
Zigbee2MQTT:debug 2020-12-09 09:37:10: Removing old log directory '/share/zigbee2mqtt/log/2020-12-09.09-36-38'
Zigbee2MQTT:debug 2020-12-09 09:37:10: Loaded state from file /share/zigbee2mqtt/state.json
Zigbee2MQTT:info  2020-12-09 09:37:10: Starting Zigbee2MQTT version 1.15.0 (commit #1.15.0)
Zigbee2MQTT:info  2020-12-09 09:37:10: Starting zigbee-herdsman...
Zigbee2MQTT:debug 2020-12-09 09:37:10: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null},"backupPath":"/share/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/share/zigbee2mqtt/database.db.backup","databasePath":"/share/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6758},"serialPort":{"path":"/dev/ttyS3"}}'
Zigbee2MQTT:error 2020-12-09 09:37:10: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2020-12-09 09:37:10: Failed to start zigbee
Zigbee2MQTT:error 2020-12-09 09:37:10: Exiting...
Zigbee2MQTT:error 2020-12-09 09:37:10: Error: Error while opening serialport 'Error: Error: I/O error setting custom baud rate of 115200'
    at Znp.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:136:32)
    at Generator.next (<anonymous>)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:23:12)
    at SerialPort.<anonymous> (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/dist/adapter/z-stack/znp/znp.js:134:49)
    at SerialPort._error (/zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:198:14)
    at /zigbee2mqtt-1.15.0/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:242:12
npm 
ERR!
code ELIFECYCLE
npm ERR! errno 1
npm 
ERR!
 zigbee2mqtt@1.15.0 start: node index.js
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.15.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.`


So, I guess it is one step further, but I still cannot say, whether this is really about a wrong baud rate or the device still cannot be found. To me the device still seems to be missing.

Thanks a lot,
Fred
